base/number.jl

```diff
@@ -1,7 +1,19 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
 ## generic operations on numbers ##
+"""
+    isinteger(x) -> Bool
+
+Test whether `x` or all its elements are numerically equal to some integer.
```

Comment:
`isinteger(x::AbstractArray)` returns a single `Bool` rather than operating elementwise, so it's not among the vectorized functions slated for deprecation in favor of compact `broadcast` syntax. Are you suggesting deprecation in favor of `all(isinteger, x)` separate from the devectorization deprecations? Best!